cmd/web-handlers.go

```diff
@@ -1406,6 +1406,7 @@ func (web *webAPIHandlers) DownloadZip(w http.ResponseWriter, r *http.Request) {
 				Name:   strings.TrimPrefix(objectName, args.Prefix),
 				Method: zip.Deflate,
 			}
+			header.ModifiedDate, header.ModifiedTime = timeToMsDosTime(info.ModTime)
```

Comment:
These are deprecated @ebozduman https://golang.org/pkg/archive/zip/#FileHeader

We should just set Modified: info.ModTime

Reply:
We are using https://github.com/klauspost/compress/blob/master/zip/struct.go#L70 and that does not have this.

Reply:
Then @klauspost perhaps should add this. Because using cryptic code to calculate ModDate is perhaps not needed.

Reply:
Actually TBH I am not sure if we need to use klauspost/compress/zip because unless @klauspost version is faster we may be better off to use compress/zip here

Or you can do what is being done at cmd/notification.go with dummyFileInfo() implementation @ebozduman